dask/dataframe/backends.py

```diff
@@ -80,6 +81,16 @@ def _(x, index=None):
 except ImportError:
     pass
 
+try:
+    import pyarrow as pa
+
+    @pyarrow_schema_dispatch.register((pd.DataFrame,))
+    def get_pyarrow_schema_pandas(obj):
+        return pa.Schema.from_pandas(obj)
+
+except ImportError:
+    pass
+
```

Comment:
I've been thinking about this a bit, and I'm pretty sure I agree with the dispatch approach you chose (rather than adding a simple classmethod to the engine) - We *may* be able to use the schema dispatch in other places.

Rather than defining this in a `try` block, can we just import pyarrow within the `get_pyarrow_schema_pandas` definition here?